lib/data_update_scripts/20210209152726_add_fastly_http_purge_feature_flag.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class AddFastlyHttpPurgeFeatureFlag
+    def run
+      FeatureFlag.add(:fastly_http_purge)
```

Comment:
I think we should get in the habit of adding/removing feature flags programmatically so that we know each Forem is aware of them

Reply:
This may be worth documenting, though we may not have any documentation on feature flags so far.

Reply:
@citizen428 where should we add it? Most feature flags are fleeting but I guess we can keep an internal list of them.

Reply:
Sorry, I meant documenting that they should be added before being enabled, not the specific flags.